pages/ksphere/kommander/1.1.0-beta/security/distributed-authnz/index.md

```diff
@@ -10,31 +10,38 @@ excerpt: Details on distributed authentication and authorization between cluster
 
 A user identity is shared across a Kommander cluster and all other provisioned clusters.
 
 ### Kommander provisioned clusters
 
-A newly provisioned cluster gets federated `kube-oidc-proxy`, `dex-k8s-authenticator` and `traefik-forward-auth` Addons. These Addons are configured to accept Kommander cluster Dex issued id tokens.
+A newly provisioned cluster gets federated `kube-oidc-proxy`, `dex-k8s-authenticator`, and `traefik-forward-auth` Addons. These Addons are configured to accept Kommander cluster Dex issued id tokens.
 
-When the `traefik-forward-auth` is used as a [Traefik ingress authenticator](https://docs.traefik.io/v1.7/configuration/backends/kubernetes/#annotations) it checks if the user identity was issued by the Kommander cluster Dex service. An anonymous user is redirected to the Kommander cluster Dex service to authenticate and confirm their identity.
+When the `traefik-forward-auth` is used as a [Traefik ingress authenticator][traefik_ingress] it checks if the user identity was issued by the Kommander cluster Dex service. An anonymous user is redirected to the Kommander cluster Dex service to authenticate and confirm their identity.
 
 Never enter your own credentials on any of the provisioned clusters. On the Kommander cluster use the static admin credentials or an external identity provider.
 
 ## Authorization
 
 There is no centralized authorization component in Kommander. Each component and service makes its own authorization decisions based on user identity.
 
-Kommander provides an [interface to federate authorization RBAC rules](../../operations/access-control/) across multiple clusters.
+Kommander provides an [interface to federate authorization RBAC rules][access_control] across multiple clusters.
 
-* The Kubernetes API server is using [RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+* The Kubernetes API server is using [RBAC Authorization][rbac_authz].
 
-* `traefik-forward-auth` applies [RBAC rules to authorize](/ksphere/konvoy/latest/security/external-idps/rbac/#portal-authorization) access based on an HTTP request. The [default Kommander workspace](../../workspaces/) comes with preconfigured Roles for accessing different operational portal services with View, Edit or Admin permissions.
+* `traefik-forward-auth` applies [RBAC rules to authorize][portal_auth] access based on an HTTP request. The [default Kommander workspace][workspaces] comes with preconfigured Roles for accessing different operational portal services with View, Edit, or Admin permissions.
 
+[access_control]: ../../operations/access-control/
 [dex_service]: https://github.com/dexidp/dex
+[kube_oidc_proxy]: https://github.com/jetstack/kube-oidc-proxy
 [oidc_id_token]: https://openid.net/specs/openid-connect-core-1_0.html#IDToken
+[portal_auth]: ../../../../konvoy/1.5/security/external-idps/rbac/#portal-authorization
```

Comment:
i still think this should be latest so that it never gets stale but we're just about to GA 1.5 so we'll be fine for awhile; can address later.

Reply:
done

Reply:
So @samvantran and @cmastr, changing to 'latest' in the testing docs breaks it when you test locally, so you can't know if you have the right path when testing before merging. That's why I've changed it here, because I don't know if it's going to work.

I suppose the linting will check to see if the link is valid or not if we use markdown links (unless we're using actual a href links, which one must do when in a paragraph, then there's no check), but, there's no way to check until after deploy if it's going to actually be the page you want. I mean, I can make sure it's latest or whatever, but, just as a QA, it's adding risk to making sure it's going to where we say it's going to. That's why this was done in this instance.